internal/clientcache/internal/daemon/search_handler.go

```diff
@@ -22,13 +23,23 @@ import (
 	"github.com/hashicorp/go-hclog"
 )
 
+type RefreshStatus string
+
+const (
+	NotRefreshing RefreshStatus = "not-refreshing"
+	Refreshing    RefreshStatus = "refreshing"
+	RefreshError  RefreshStatus = "refresh-error"
+)
```

Comment:
Mind adding comments to these exported variables?

Reply:
Done.